torch/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ set(TORCH_PYTHON_SRCS
 # NB: This has to match the condition under which the JIT test directory
 #     is included (at the time of writing that's in caffe2/CMakeLists.txt).
 if (BUILD_TEST AND NOT MSVC AND NOT USE_ROCM)
+    add_definitions(-DBUILDING_TESTS)
```

Comment:
What does this do ?

Reply:
It adds a preprocessor macro `BUILDING_TESTS`